dactyl-config.yml

```diff
@@ -1594,6 +1594,10 @@ pages:
         targets:
             - en
             - ja
+    -   md: tutorials/get-started/get-started-using-java.md
+        targets:
+            - en
+            - ja
```

Comment:
I don't have a ja version of this, but it doesn't look like there is a ja translation for python either. Is it ok to keep this until we get a translation?

Reply:
Yes, our standard practice is to leave it like this until we have a translation available. The build process automatically adds an apology in Japanese to the top of any pages that aren't translated.